sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1979,6 +1979,13 @@ object SQLConf {
       .doc("When true, the ArrayExists will follow the three-valued boolean logic.")
       .booleanConf
       .createWithDefault(true)
+
+  private[spark] val CENTRAL_REPOSITORY =
+    ConfigBuilder("spark.sql.centralRepository")
+      .doc("The default central repository used for downloading Hive jars " +
+        "in IsolatedClientLoader.")
+      .stringConf
+      .createWithDefault("https://repo1.maven.org/maven2")
```

Comment:
We do not need to provide a default value here. Without any value, it will use maven central, I think. 

In the Hive tests, we can set the conf to some default repo in TestHiveContext? See this link https://www.deps.co/guides/public-maven-repositories/ We can use google mirror for avoiding we are blocked by maven central.

Reply:
If so, then shouldn't this be called something like "additional repo" here and below?

Reply:
If provided, I think it will try the provided mirror first and then central? 

In general, it is weird to hardcode the default mirror.

Reply:
Yes you are right; see `SparkSubmitUtils.buildIvySettings`. It's an additional "remote repo", which is used in addition to central. That's why I'm saying this should not be called "central repo", as it won't (and shouldn't) override resolving against Maven Central -- almost nothing would work without that. This looks good if we can clarify the semantics in the naming and description. The default should be "None" here

Reply:
That's right, I also test locally without setting any additional remote repo, it will pass.
Change the default value and set the config to google mirror in 49ea1cd.